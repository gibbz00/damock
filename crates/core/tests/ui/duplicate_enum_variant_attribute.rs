#[derive(mock_default::Mock)]
enum Foo {
    #[mock]
    Bar,
    #[mock]
    Baz,
}

fn main() {}
