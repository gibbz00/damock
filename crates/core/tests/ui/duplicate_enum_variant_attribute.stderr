error: expected only one #[mock] enum variant attribute, unable to infer which one to use.
 --> tests/ui/duplicate_enum_variant_attribute.rs:2:1
  |
2 | enum Foo {
  | ^^^^
